src/styles/forms.css

```diff
@@ -211,3 +211,25 @@ label {
 .error-message a {
   text-decoration: underline;
 }
+
+.login-container {
+  height: 70vh;
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  justify-content: center;
+}
+
+.finding-error-container {
+  margin: 0 auto;
+  text-align: center;
+  width: 50vw;
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  background-color: var(--secondary-background-purple);
+  border-radius: 8px;
+  border:1px solid var(--header-background-purple);
+  padding:20px;
+  box-shadow: 0px 4px 10px rgba(0,0,0,0.5);
+}
```

Comment:
Is it possible to just utilize the className and styles that are applied to the form? e.g. from the Form module, `Form`

Reply:
You can also apply the `centered-text` class name

Reply:
Totally, Doesn't make sense to have duplicate classes, I changed it :) ✅